registration/tests/utils.py

```diff
@@ -65,10 +84,12 @@ def form(self):
         """
         return self.client.find_element_by_tag_name('form')
 
-    def fill_form(self, form_data):
+    def fill_form(self, form_data, validate_fields=None):
         """
         Fill in the form with the given data.
         """
+        if validate_fields is None:
+            validate_fields = tuple()
```

Comment:
Nit:
```python
validate_fields = validate_fields or ()
```

I personally find this notation much more concise.

Reply:
:+1: